### DISCUSSION TITLE
Issues with GitHub Classroom

### DISCUSSION THREAD

**Discussion by Kris-Luo-98** [community/community#61725] [GitHub Education]
### Select Topic Area

Question

### Body

Hi everyone,

I am currently using GitHub Classroom to distribute assignments. I don't know why some of my students are able to successfully clone my template repository, while others are not. Has anyone encountered this issue before? Could it be related to the number of students? The roster contains 700 names.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Kris-Luo-98 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
